main.qml

```diff
@@ -859,7 +859,9 @@ ApplicationWindow {
                     ", priority: ", priority,
                     ", description: ", description);
 
-        showProcessingSplash("Creating transaction");
+        var splashMsg = qsTr("Creating transaction...");
+        splashMsg += appWindow.currentWallet.isLedger() ? qsTr("\n\nPlease check your hardware wallet –\nyour input may be required.") : "";
```

Comment:
Do we need to end the sentence with a period here?

Reply:
I'd be fine with or without period.

Reply:
I think it looks better with a period, but I don’t care really.